README.md

```diff
@@ -1,6 +1,5 @@
 # CacheAdvance
 
-[![CI Status](https://travis-ci.com/dfed/CacheAdvance.svg?branch=main)](https://travis-ci.com/dfed/CacheAdvance)
```

Comment:
it's possible that we can get this working with GitHub Actions but I'm not convinced this badge has ever been super helpful